dev/build.example_fat/publish/servers/FATServer/server.xml

```diff
@@ -13,6 +13,8 @@
     <featureManager>
         <feature>servlet-3.1</feature>
         <feature>componenttest-1.0</feature>
+        <feature>wsbytebuffer-1.0</feature>
+        <feature>osgiconsole-1.0</feature>
```

Comment:
The console would need to come out - but also - I don't think we really need a feature at all and we should not need one here.